### PR TITLE
Use CLDR 29

### LINKED ISSUE #404: Upgrade to CLDR 29
Here we go again...

http://blog.unicode.org/2016/03/cldr-version-29-released.html
http://www.unicode.org/reports/tr35/tr35-43/tr35.html

### PR DESCRIPTION
**Pull request by akx**
Everything... seems to work, at least!

Fixes #404

### CODE REVIEW

File: tests/test_day_periods.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     ("de", time(7, 42), "morning1"),  # (from, before)
     ("de", time(3, 11), "night1"),  # (after, before)
     ("fi", time(0), "midnight"),  # (at)
-    ("en_US", time(12), "noon"),  # (at)
+    ("en", time(12), "noon"),  # (at)
```
Comment (jtwang) [inline tests/test_day_periods.py]: Why did this change? Purely to make the tests consistent?

**Comment by jtwang** [comment]
Thanks for tackling this! I expected there to be waaaay more changes, it's awesome that there aren't! I am kind of bummed that zh_Hant is still missing am/pm from it's abbreviated day period definitions. :P

**Comment by jtwang** [comment]
ack dc449a1c522bda4d84ab3a01ac1dc9ca8e8bf06b, 09acaaa2e6737f2d03f4c8371e81760357d89df5

(although more test coverage would be awesome)

File: tests/test_day_periods.py (hunk 1 above)
Comment (akx) [inline tests/test_day_periods.py]: Good point. I'm actually not sure why I changed that...

**Comment by akx** [comment]
@jtwang Rebased and fixed that en_US thing.

**Comment by jtwang** [comment]
ack 34c92cb220a6eeac40f1cb5b60241ddffca6c3ff

Looks good, merge! (What's the worst that can happen ;)

**Pull request merged**